### PR TITLE
style: `pl-variable-output` style fixes

### LINKED ISSUE #4248: pl-variable-output: label as Octave/Matlab
Would you accept a patch to change the label from Matlab to Octave/Matlab (or Matlab/Octave) if you prefer?

Or maybe the labels could be customizable?  As sometimes Octave dev, I'd be happier to be advertising Octave to my students!

### LINKED ISSUE #7915: pl-variable-output inner tags are not prefixed
See title. Inner tags should be `pl-variable` and not just `variable`. Docs and example questions should be updated accordingly (but allow for backward compatibility).

### PR DESCRIPTION
**Pull request by eliotwrobson**
Resolves #4248 and resolves #7915. Also updates docs and example course accordingly. cc @cbm755

### CODE REVIEW

File: apps/prairielearn/elements/pl-variable-output/pl-variable-output.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     r_data = ""
     sympy_data = "from sympy import *\n\n"
     for child in element:
-        if child.tag == "variable":
+        if child.tag in {"variable", "pl-variable"}:
```
Comment (jonatanschroeder) [inline apps/prairielearn/elements/pl-variable-output/pl-variable-output.py:94]: Maybe add a comment that `variable` is accepted for backwards compatibility?

File: docs/elements.md [hunk 2]
```diff
@@ -1505,17 +1505,17 @@ def generate(data):
...
-| `show-mathematica` | boolean | true     | Toggles the display of the Mathematica tab.    |
-| `show-python`      | boolean | true     | Toggles the display of the Python tab.         |
-| `show-r`           | boolean | true     | Toggles the display of the R tab.              |
-| `show-sympy`       | boolean | true     | Toggles the display of the SymPy tab.          |
+| Attribute          | Type    | Default  | Description                                                              |
+| ------------------ | ------- | -------- | ------------------------------------------------------------------------ |
+| `digits`           | integer | —        | Number of digits to display after the decimal.                           |
+| `default-tab`      | string  | `matlab` | Select the active tab.                                                   |
+| `show-matlab`      | boolean | true     | Toggles the display of the Matlab tab.                                   |
+| `show-mathematica` | boolean | true     | Toggles the display of the Mathematica tab. Also compatible with Octave. |
```
Comment (jonatanschroeder) [inline docs/elements.md:1513]: Is the comment in the wrong row?

Comment (eliotwrobson) [reply docs/elements.md:1513]: Not the first time I've mixed up Matlab and Mathematica 🤦‍♂️

File: apps/prairielearn/elements/pl-variable-output/pl-variable-output.py (hunk 1 above)
Comment (eliotwrobson) [reply apps/prairielearn/elements/pl-variable-output/pl-variable-output.py:95]: Done here https://github.com/PrairieLearn/PrairieLearn/pull/7916/commits/ee958df312e9267177bbaa86510c2060a1816cc3

File: docs/elements.md (hunk 2 above)
Comment (eliotwrobson) [reply docs/elements.md]: Fixed here https://github.com/PrairieLearn/PrairieLearn/pull/7916/commits/ee958df312e9267177bbaa86510c2060a1816cc3

**Pull request merged**